tests/Agent/IntegrationTests/IntegrationTestHelpers/RemoteServiceFixtures/RemoteApplicationFixture.cs

```diff
@@ -564,6 +564,19 @@ protected void GetAndAssertSuccessStatus(string address, bool expectedSuccessSta
                 }
             }
         }
+
+        protected void PostString(string address, string payload)
```

Comment:
Because this is posting a json string in a format specifically for an Azure Function, should it be moved to one of the azure function fixtures?

Reply:
Good catch. Refactored.